Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pe/FileHeader.java

```diff
@@ -313,7 +313,16 @@ void processSections(OptionalHeader optHeader) throws IOException {
 	            int sizeOfRawData = sectionHeaders[i].getSizeOfRawData();
 	    		sizeOfRawData  = PortableExecutable.computeAlignment(sizeOfRawData, optHeader.getFileAlignment());
 	    		sectionHeaders[i].setSizeOfRawData(sizeOfRawData);
-	    		
+
+	            int virtualSize = sectionHeaders[i].getVirtualSize();
+	    		virtualSize  = PortableExecutable.computeAlignment(virtualSize, optHeader.getSectionAlignment());
+	    		/* Truncate virtualSize to sizeOfRawData to reduce memory blocks of uninitialized data.
+	    		 * Note: This may still cause problems if code interacts with memory beyond sizeOfRawData but before virtualSize.
+	    		 *       So you can remove the following line to get the complete memory range as would be loaded by the Windows PE loader.
+	    		 */
+	    		virtualSize = virtualSize > sizeOfRawData ? sizeOfRawData : virtualSize; 
```

Comment:
I'm still losing the .imrsiv section of explorer.exe.  Did you mean to continue to set virtualSize to 0 here when sizeOfRawData is 0?  The section will still have a virtual size associated with it despite none of its bytes coming from the file.

Reply:
No. This change was included by accident when I rebased into a single commit.
I'm terribly sorry.
It should now be fixed. I just checked again.